### ISSUE TITLE
Release 0.2.1 with FluxibleMixin export

### ISSUE THREAD

**Issue by cesarandreu** [yahoo/fluxible#40]
Would be nice to release this: https://github.com/yahoo/fluxible/commit/9d1f17c848cd967993bda71d094e81718b7c7e23

I actually considered making the suggestion for adding a Mixin alias as FluxibleMixin when I tried out 0.2.0, but I thought it would be such a minor thing that it wasn't worth it. But seeing as it's already been merged, it'd be great if it were released.

Basically, i means I can replace this:
```js
var {Mixin: FluxibleMixin} = require('fluxible');
```

With this:
```js
var {FluxibleMixin} = require('fluxible');
```

**Comment by redonkulus** [comment]
@cesarandreu this has been released in [0.2.1](https://www.npmjs.com/package/fluxible).

**Comment by cesarandreu** [comment]
Thanks!
